internal/auth0/connection/resource_clients_test.go

```diff
@@ -96,15 +96,6 @@ resource "auth0_connection_clients" "my_conn_client_assoc" {
 }
 `
 
-const testAccConnectionClientsWithNoEnabledClients = givenASingleConnection + `
```

Comment:
This was a bit unnecessary as it's exactly the first scenario with minimal config. So I swapped it out with giving just the single connection scenario making the association resource get deleted.